eventhubs/client/README.md

```diff
@@ -0,0 +1,192 @@
+#### AMQP exception
+This is a general exception for AMQP related failures, which includes the AMQP errors as ErrorCondition and the context 
+that caused this exception as ErrorContext. 'isTransient' is a boolean indicating if the exception is a transient error
+or not. If true, then the request can be retried; otherwise not.
+
+- ErrorCondition: it contains constants common to the AMQP protocol and constants shared by Azure services. More detail
+  can be found in the link: [Event Hubs Messaging Exceptions][event_hubs_messaging_exceptions].
+- ErrorContext, it provides context that caused the AmqpException. The error occurs could be from AmqpConnection, 
+  AmqpSession, or AmqpLink. Such as SessionErrorContext and LinkErrorContext, the context for an error that occurs in an
+  AMQP session and AMQP link, respectively.
+
+The recommended way to solve the specific exception the AMQP exception represents is to follow the [Event Hubs Messaging Exceptions][event_hubs_messaging_exceptions] guidance.
+
+#### Operation cancelled exception
+It occurs when the underlying AMQP layer encounters an abnormal link abort or the connection is disconnected in an 
+unexpected fashion. It is recommended to attempt to verify the current state and retry if necessary.
+
+#### Message size exceeded
+Event data, both individual and in batches, have a maximum size allowed. This includes the data of the event, as well as
+any associated metadata and system overhead. The best approach for resolving this error is to reduce the number of events
+being sent in a batch or the size of data included in the message. Because size limits are subject to change, please 
+refer to Azure Event Hubs quotas and limits for specifics.
+
+
+### Other exceptions
+For detailed information about these and other exceptions that may occur, please refer to 
+[Event Hubs Messaging Exceptions][event_hubs_messaging_exceptions].
+
+## Next steps
+Beyond those discussed, the Azure Event Hubs client library offers support for 
+many additional scenarios to help take advantage of the full feature set of the Azure Event Hubs service. In order to help explore some of the these scenarios, the following set of sample is available:
+- [Inspect Event Hub and partition properties][sample_get_event_hubs_metadata]
+- [Publish an event to an Event Hub][sample_send_event]
+- [Publish events to a specific Event Hub partition with producer option][sample_send_producer_option]
+- [Publish events to a specific Event Hub partition with send option][sample_send_send_option]
+- [Publish events with custom metadata][sample_send_custom_event_data]
+- [Consume events from an Event Hub partition][sample_receive_event]
+- [Consume event batch][sample_receive_batch]
+- [Save the last read event and resume from that point][sample_sequence_number]
+
+## Contributing
+
+If you would like to become an active contributor to this project please refer to our [Contribution Guidelines](./../../CONTRIBUTING.md) for more information.
```

Comment:
There is a contributing.md in the current directory. `./CONTRIBUTING.md` should work.

Reply:
I don't find the file in the current directory.

Reply:
Have you pulled latest from master? https://github.com/Azure/azure-sdk-for-java/tree/master/eventhubs/client